<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgInvestmentsRanking</class>
 <widget class="QWidget" name="wdgInvestmentsRanking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>524</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Investments ranking</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tab">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_">
        <attribute name="icon">
         <iconset resource="../images/xulpymoney.qrc">
          <normaloff>:/xulpymoney/crown.png</normaloff>:/xulpymoney/crown.png</iconset>
        </attribute>
        <attribute name="title">
         <string>With same product merging current operations</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="mqtwObjects" name="mqtwCurrentOperations" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="icon">
         <iconset resource="../images/xulpymoney.qrc">
          <normaloff>:/xulpymoney/crown.png</normaloff>:/xulpymoney/crown.png</iconset>
        </attribute>
        <attribute name="title">
         <string>With same product merging operations</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="mqtwObjects" name="mqtwOperations" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionSameProduct">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/bundle.png</normaloff>:/xulpymoney/bundle.png</iconset>
   </property>
   <property name="text">
    <string>Same product Investments merging current operations</string>
   </property>
   <property name="toolTip">
    <string>Same product Investments merging current operations</string>
   </property>
  </action>
  <action name="actionSameProductFIFO">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/bundle.png</normaloff>:/xulpymoney/bundle.png</iconset>
   </property>
   <property name="text">
    <string>Same product Investments merging operations</string>
   </property>
   <property name="toolTip">
    <string>Same product Investments merging operations</string>
   </property>
  </action>
  <action name="actionProduct">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/books.png</normaloff>:/xulpymoney/books.png</iconset>
   </property>
   <property name="text">
    <string>Product report</string>
   </property>
   <property name="toolTip">
    <string>Product report</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mqtwObjects</class>
   <extends>QWidget</extends>
   <header location="global">myqtablewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
